robottelo/ui/login.py

```diff
@@ -38,8 +39,24 @@ def logout(self):
         self.wait_for_ajax()
 
     def is_logged(self):
-        """Checks whether an user is logged"""
-        if self.wait_until_element(locators["login.gravatar"]):
+        """Checks whether an user is logged by validating the session cookie"""
+        cookies = {}
+        wd_cookies = self.browser.get_cookies()
+        # reformatting the cookies for requests.get request
+        for cookie in wd_cookies:
+            cookies[cookie['name']] = cookie['value']
```

Comment:
The above several lines of code can be compacted down to this:

```python
cookies = {
    cookie['name']: cookie['value']
    for cookie in self.browser.get_cookies()
}
```

Or, if we care about Python 2.6 compatibility:

```python
cookies = dict(
    (cookie['name'], cookie['value'])
    for cookie in self.browser.get_cookies()
)
```